IdentityCore/src/webview/MSIDWebviewAuthorization.m

```diff
@@ -83,7 +83,7 @@ + (void)startSession:(MSIDWebviewSession *)session
     // check session nil
     if (!session)
     {
-        NSError *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInvalidRequest, @"Interactive session failed to create.", nil, nil, nil, context.correlationId, nil);
+        NSError *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorInvalidDeveloperParameter, @"Interactive session failed to create.", nil, nil, nil, context.correlationId, nil);
```

Comment:
why is this developer parameter? This feels like it would be MSAL internal error or system error :)

Reply:
It is an argument to the method itself. But I do understand where the argument is coming from - this method is only called by other methods that *should* create a session.
Thus, changing it to MSIDErrorInvalidInternalParameter.